content/models.py

```diff
@@ -3,27 +3,49 @@
 
 
 class OurStory(models.Model):
-    img = models.ImageField(blank=True, null=True)
-    text = models.TextField()
-    color = models.CharField(max_length=20, blank=True)
-    order = models.PositiveIntegerField()
+	img = models.ImageField(upload_to='aboutus', blank=True, null=True)
+	text = models.TextField()
+	color = models.CharField(max_length=20, blank=True)
+	order = models.PositiveIntegerField()
 
 
 
-    def __str__(self):
-        return self.text
+	def __str__(self):
+		return self.text
 
 
 class Ourteam(models.Model):
-    img = models.ImageField(upload_to='uploads')
-    text = models.TextField()
-    us_team = models.CharField(max_length=100)
-    peru_team = models.CharField(max_length=100)
-    board_team = models.CharField(max_length=100)
+	img = models.ImageField(upload_to='aboutus', blank=True, null=True)
```

Comment:
just want to verify that both `OurStory` and `Ourteam` are being uploaded to the same place.